src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.cs

```diff
@@ -338,45 +409,87 @@ private static void EnsureBuffer(ReadOnlySpan<byte> buffer, int requiredLength)
             }
         }
 
-        private List<(int offset, int length)> AcquireRangeList()
+        private readonly struct StackFrame
         {
-            List<(int offset, int length)>? ranges = Interlocked.Exchange(ref _rangeListAllocation, null);
-
-            if (ranges != null)
+            public StackFrame(CborMajorType type, int frameOffset, int? remainingDataItems,
+                              int? currentKeyOffset, bool currentItemIsKey,
+                              (int Offset, int Length)? previousKeyRange, HashSet<(int Offset, int Length)>? previousKeyRanges)
             {
```

Comment:
Indent, don't align. 

```suggestion
            public StackFrame(CborMajorType type, int frameOffset, int? remainingDataItems,
                int? currentKeyOffset, bool currentItemIsKey,
                (int Offset, int Length)? previousKeyRange, HashSet<(int Offset, int Length)>? previousKeyRanges)
            {
```

(And while we don't quite have a style of chop v wrap; I feel like chop is predominant, and makes future edits easier to review)

```suggestion
            public StackFrame(
                CborMajorType type,
                int frameOffset,
                int? remainingDataItems,
                int? currentKeyOffset,
                bool currentItemIsKey,
                (int Offset, int Length)? previousKeyRange,
                HashSet<(int Offset, int Length)>? previousKeyRanges)
            {
```